src/algebra/direct_sum/module.lean

```diff
@@ -301,6 +309,19 @@ lemma submodule_is_internal_iff_independent_and_supr_eq_top (A : ι → submodul
 ⟨λ i, ⟨i.independent, i.supr_eq_top⟩,
  and.rec submodule_is_internal_of_independent_of_supr_eq_top⟩
 
+/-- If a collection of submodules has just two indices, `i` and `j`, then
+`direct_sum.submodule_is_internal` is equivalent to `is_compl`. -/
+lemma submodule_is_internal_iff_is_compl (A : ι → submodule R M) {i j : ι} (hij : i ≠ j)
+  (h : (set.univ : set ι) = {i, j}) :
+  submodule_is_internal A ↔ is_compl (A i) (A j) :=
+begin
+  have : ∀ k, k = i ∨ k = j := λ k, by simpa using set.ext_iff.mp h k,
```

Comment:
I'm not really sure which of `∀ k, k = i ∨ k = j` or `(set.univ : set ι) = {i, j}` or `(finset.univ : set ι) = {i, j}` is the better spelling.

Reply:
I'm also not sure. I say we see how we get on with what you've proposed.